build-system/common/utils.js

```diff
@@ -89,7 +89,7 @@ function logFiles(files) {
  */
 function getFilesToCheck(globs, options = {}) {
   if (argv.files) {
-    return logFiles(globby.sync(argv.files.split(',')));
+    return logFiles(globby.sync(argv.files.split(',').map((s) => s.trim())));
```

Comment:
I think this should be implemented as a helper function in `build-system/common/utils.js`:
```js
/**
 * Extracts the list of files from argv.files.
 *
 * @return {Array<string>}
 */
function getFilesFromArgv() {
  return argv.files
    ? globby.sync(argv.files.split(',').map((s) => s.trim()))
    : [];
}
```

Then, you can do this in `getFilesToCheck()`...
```js
function getFilesToCheck(globs, options = {}) {
  if (argv.files) {
    return logFiles(getFilesFromArgv());
  }
  // ...
}
```

... and use `getFilesFromArgv()` in all the other places in this PR.

Reply:
Done.